README.md

```diff
@@ -25,8 +25,6 @@ sudo apt-get install nodejs cmake build-essential swig python3.8
 sudo npm install --global npm
```

Comment:
"change x by y" block fails with output:
> VARIABLE must be number

<img width="155" alt="image" src="https://user-images.githubusercontent.com/126523/221427083-072d4322-3a6f-4530-92bb-2297efb9064f.png">

```xml
<xml xmlns="http://www.w3.org/1999/xhtml">
  <variables>
    <variable type="" id=";/fOvI=r,rhuvz3kg:!y" islocal="false" iscloud="false">i</variable>
    <variable type="" id="mXKmvt*Z7vJ|3:~+HU:#" islocal="false" iscloud="false">j</variable>
    <variable type="" id="2ogUd*aq6_gfGWacz#/2" islocal="false" iscloud="false">testvar</variable>
    <variable type="" id="q=qhREa7eU=jdV,%uUs-" islocal="false" iscloud="false">k</variable>
  </variables>
  <block type="control_run" id="0fNAs-vY8^BL;[T$]eav" x="159" y="8">
    <next>
      <block type="data_setvariableto" id="(RpP{VpAeA!KNbm}Tscv">
        <field name="VARIABLE" id="2ogUd*aq6_gfGWacz#/2" variabletype="">testvar</field>
        <value name="VALUE">
          <shadow type="text" id="2#bAQ,?kgq|Jf~}kEU$P">
            <field name="TEXT">4</field>
          </shadow>
        </value>
        <next>
          <block type="data_changevariableby" id="Ed+~U!%,|,PK^ef%1JHG">
            <field name="VARIABLE" id="2ogUd*aq6_gfGWacz#/2" variabletype="">testvar</field>
            <value name="VALUE">
              <shadow type="math_number" id=";T;+tWn,?Zbt}mK,j/0C">
                <field name="NUM">1</field>
              </shadow>
            </value>
          </block>
        </next>
      </block>
    </next>
  </block>
</xml>
```

Reply:
Re-opening. Still failing with the same error. I noticed that it fails with the program above, where the variable is set to "4", but not when the variable is set to a function returning a number, like `gmpc()`